.travis.yml

```diff
@@ -1,216 +0,0 @@
-# This Travis job script has been generated by a script via
```

Comment:
Now that this `.travis.yml` file has been removed, would it be worth migrating the fixtures in the test suite from Travis-style YAML files to GHA-style YAML files? I did this in my version of the GHA backend, and I wonder if that would be the right thing going forward.

Reply:
I'm not sure what to do. Technically I don't want to drop Travis support. But if we (`haskell-ci` org) run out of credits, so we cannot self test, it's a bit of a problem.

But in general, yes. We should test all backends across all fixtures. The test-suite might need some reorganization to not make it all a complete mess :)

Reply:
> But in general, yes. We should test all backends across all fixtures. The test-suite might need some reorganization to not make it all a complete mess :)

In my `wip/github-actions` branch, I include both `.travis.yml` and `github-actions.yml` files for each test fixture (see [this commit](https://github.com/RyanGlScott/haskell-ci/commit/00d793b965298c57774174c5ccbcc1c805a6ae91)). Is this what you had in mind?

Reply:
Yes.

And you can see yourself `fixtures` directory is complete mess. Doesn't invite to add any more tests :)

Reply:
OK. I was going to ask if you'd like me to create a PR to add these `github-actions.yml` files to the test suite. Perhaps you'd prefer to wait until the test suite is in a cleaner state first?

Reply:
Yeah. I have a thing in mind which would allow making fixtures situation a bit nicer. Will do that soon before I forget that.